### DISCUSSION TITLE
Can Charliecloud run on CentOS 6.10?

### DISCUSSION THREAD

**Discussion by natbutter** [hpc/charliecloud#1653] [General]
We have an ageing HPC system currently running CentOS 6.10, many new applications can no longer run on it. Singularity has pushed the lifetime of the system, but now trying to run something modern like Matlab 2021a in a container will fail when it wants to use system calls (i.e. getrandom) that do not exist on the old Host kernel. 

Was hoping Charliecloud could give us another few version iterations of life. My attempts at installing it in a Virtual Box is proving difficult, so before I go chasing various errors was hoping to see if anyone had any insight?

Cheers

**Comment by reidpr** [comment]
Hello @natbutter, unfortunately CentOS 6’s kernel is too old. If you upgrade the kernel, it should work; whether that’s worth your time vs. just upgrading the whole OS would have to be your call.

**Reply by natbutter** [reply]
Thanks for clarifying.
